vendor/engines/bulk_email/app/mailers/bulk_email/mailer.rb

```diff
@@ -2,16 +2,24 @@ module BulkEmail
 
   class Mailer < BaseMailer
 
-    def send_mail(recipient:, custom_subject:, facility:, custom_message:)
+    def send_mail(recipient:, custom_subject:, facility:, custom_message:, subject_product_id: nil)
```

Comment:
I had meant it in the sense that the product is the subject of the bulk email run, but the param name isn't really clear.

This does use ActiveJob with globalid, so it could just be `product`.